includes/api/class-wc-rest-product-variations-controller.php

```diff
@@ -39,6 +39,20 @@ class WC_REST_Product_Variations_Controller extends WC_REST_Products_Controller
 	 */
 	protected $post_type = 'product_variation';
 
+	/**
+	 * Name of parent product table in SQL query.
+	 *
+	 * @var string
+	 */
+	protected $parent_product_table_name = 'p_wc_variation_parent';
```

Comment:
Seems like this is not used in any place. Probably just need to remove.

Reply:
Good catch. Removed